src/prompts/ui.tsx

```diff
@@ -224,6 +281,7 @@ export default class Ui extends Component<UiProps, UiState> {
                     />
                 );
             case 'alert':
+            case 'banner':
                 return (
                     <Alert
```

Comment:
Semantics of naming, any better name for Alert if it can present the banners too?

Maybe there isn't, just a thought.

Reply:
Renamed comp to `<Dialog />`